uportal-war/src/main/java/org/jasig/portal/security/xslt/IXalanMessageHelper.java

```diff
@@ -38,4 +38,12 @@
     public String getMessage(String code, String language, String arg1, String arg2);
     
     public String getMessage(String code, String language, String arg1, String arg2, String arg3);
+
+    public String getMessageForEmacsScript(String code, String language);
+
+    public String getMessageForEmacsScript(String code, String language, String arg1);
+
+    public String getMessageForEmacsScript(String code, String language, String arg1, String arg2);
+
+    public String getMessageForEmacsScript(String code, String language, String arg1, String arg2, String arg3);
```

Comment:
Yes only first is used, but it was to match the signature to others methods as i seen others used (not all)...